bentoml/service.py

```diff
@@ -533,6 +556,11 @@ class BentoService:
     # See `web_static_content` function above for more
     _web_static_content = None
 
+    # Labels of this BentoService. Labels intended for identifying attributes of
+    # BentoService that are meaningful and relevant to users. Each label is a key-value
+    # pair. Each label's key must be unique for the given BentoService.
+    _labels = None
```

Comment:
remove this? Since this is only a YataiService concept as dicussed

Reply:
The idea is that, if I'm just using `save_to_dir` and build my own model management layer, this _labels here does not make sense